docs/platforms/javascript/common/tracing/instrumentation/index.mdx

```diff
@@ -227,32 +167,23 @@ if (span) {
 
 ### Adding attributes to all spans
 
-To add an attribute to all spans, use the `beforeSendTransaction` callback:
+To add an attribute to all spans, use the `beforeSendSpan` callback:
 
 ```javascript
 Sentry.init({
   // dsn, ...
-  beforeSendTransaction(event) {
-
-    // set the attribute on the root span
-    event.contexts.trace.data = {
-      ...event.contexts.trace.data,
-      myAttribute: "myValue",
-    }
-
-    // and on all child spans
-    event.spans.forEach(span => {
-      span.data = {
-        ...span.data,
-        myAttribute: "myValue",
-      }
-    });
-  }
+  beforeSendSpan(span) {
+    span.data = {
+      ...span.data,
+      "app.version": "1.2.3",
+      "environment.region": "us-west-2",
```

Comment:
l: let's avoid promoting something like environment and version just yet. I asked internally again what our plan is for stuff like `release` 

```suggestion
      "environment.region": "us-west-2",
```